core/src/com/unciv/ui/victoryscreen/VictoryScreen.kt

```diff
@@ -54,24 +54,30 @@ class VictoryScreen(val worldScreen: WorldScreen) : PickerScreen() {
         val playerVictoryType = playerCivInfo.victoryManager.hasWonVictoryType()
         if (playerVictoryType != null) {
             someoneHasWon = true
-            when (playerVictoryType) {
-                VictoryType.Cultural -> wonOrLost("You have won a cultural victory!")
-                VictoryType.Domination -> wonOrLost("You have won a domination victory!")
-                VictoryType.Scientific -> wonOrLost("You have won a scientific victory!")
-                VictoryType.Neutral -> wonOrLost("You have won!")
-            }
+            wonOrLost(
+                when (playerVictoryType) {
+                    VictoryType.Cultural -> "You have won a cultural victory!"
+                    VictoryType.Domination -> "You have won a domination victory!"
+                    VictoryType.Scientific -> "You have won a scientific victory!"
+                    VictoryType.Diplomatic -> "You have won a diplomatic victory!"
+                    VictoryType.Neutral -> "You have won!"
+                }
+            )
         }
         for (civ in gameInfo.civilizations.filter { it.isMajorCiv() && it != playerCivInfo }) {
             val civVictoryType = civ.victoryManager.hasWonVictoryType()
             if (civVictoryType != null) {
                 someoneHasWon = true
                 val winningCivName = civ.civName
-                when (civVictoryType) {
-                    VictoryType.Cultural -> wonOrLost("[$winningCivName] has won a cultural victory!")
-                    VictoryType.Domination -> wonOrLost("[$winningCivName] has won a domination victory!")
-                    VictoryType.Scientific -> wonOrLost("[$winningCivName] has  won a scientific victory!")
-                    VictoryType.Neutral -> wonOrLost("[$winningCivName] has won!")
-                }
+                wonOrLost(
+                    when (civVictoryType) {
+                        VictoryType.Cultural -> "[$winningCivName] has won a cultural victory!"
+                        VictoryType.Domination -> "[$winningCivName] has won a domination victory!"
+                        VictoryType.Scientific -> "[$winningCivName] has won a scientific victory!"
+                        VictoryType.Diplomatic -> "[$winningCivName] has won a diplomatic victory!"
```

Comment:
These should probably be replaced with a "has won a [$civVictoryType] victory!" template